resources/aws_ec2.go

```diff
@@ -369,7 +369,6 @@ func (obj *AwsEc2Res) longpollWatch() error {
 				}
 			}
 			if obj.State == "terminated" {
-				obj.client.WaitUntilInstanceExistsWithContext(ctx, diInput)
```

Comment:
Can you tell me why?

Even better if this is in the body of the commit message.